### ISSUE TITLE
No right click in file chooser

### ISSUE THREAD

**Issue by lah7** [lah7/gtk3-classic#33]
### Bug Description
Right clicking an item in the file chooser does nothing. The following error appears in the Terminal:

```
(geany:47418): Gtk-CRITICAL **: 21:03:33.896: gtk_popover_set_pointing_to: assertion 'GTK_IS_POPOVER (popover)' failed

(geany:47418): Gtk-CRITICAL **: 21:03:33.896: gtk_popover_popup: assertion 'GTK_IS_POPOVER (popover)' failed
```

### Expected Behaviour

A traditional context menu appears with right click options.

### Screenshots

![Screenshot_20210321_210312](https://user-images.githubusercontent.com/13032135/111920846-350b0780-8a89-11eb-9c50-35ba0695b208.png)

### Environment
 - OS: EndeavourOS
 - Desktop Environment: KDE
 - gtk3-classic Version: 3.24.27

**Comment by N0rbert** [comment]
I'm not familiar with Gtk codebase. 
Here is what I see in `~/.xsession-errors` with mentioned [`popovers__file-chooser-list.patch`](https://github.com/lah7/gtk3-classic/blob/backport_ubuntu_focal_3.24.20/popovers__file-chooser-list.patch) from https://github.com/lah7/gtk3-classic/tree/backport_ubuntu_focal_3.24.20 branch.

When I start Firefox, then `<Ctrl>`+`<O>`, do right mouse click in biggest white area, I see: 

```
(firefox:1847): Gtk-CRITICAL **: hh:mm:ss.706: gtk_popover_set_pointing_to: assertion 'GTK_IS_POPOVER (popover)' failed
(firefox:1847): Gtk-CRITICAL **: hh:mm:ss.708: gtk_popover_popup: assertion 'GTK_IS_POPOVER (popover)' failed
```

As temporary solution I have disabled this patch in [my PPA](https://launchpad.net/~nrbrtx/+archive/ubuntu/gtk3-classic-test) for Ubuntu MATE 20.04 LTS.

We may try to ping @jonathonf , @TomaszGasior or maybe @luigifab as [similar PPA owner](https://launchpad.net/~luigifab/+archive/ubuntu/packages-gtk3-classic) to get some help about this patch.

**Comment by luigifab** [comment]
Interesting, I already saw that a right click do nothing, but not searched why (for me: debian testing + gtk 3.24.23).
When this patch is disabled, the popover works?

**Comment by lah7** [comment]
The patch works in 3.24.30. I originally fixed it in ed8470be4939999902fd643d8c281619dd31add1 for 3.24.29. Like before, the code that actually opens the menu was missing and caused that error. For 3.24.**20**, it's probably being patched at the wrong lines and will need amending in that branch.

**Comment by luigifab** [comment]
I recompiled for me (debian testing + gtk 3.24.23) with the updated [popovers__file-chooser-list.patch](https://github.com/lah7/gtk3-classic/blob/master/popovers__file-chooser-list.patch) and with [treeview__alternating_row_colours.patch](https://github.com/lah7/gtk3-classic/blob/master/treeview__alternating_row_colours.patch) and with [notebook_wheel_scroll.patch](https://github.com/lah7/gtk3-classic/blob/master/notebook_wheel_scroll.patch) (I also updated my PPA until I remove it).
The initial issue is fixed.

**Comment by lah7** [comment]
I see what's happening but I'm really struggling with `quilt` to fix it for 20.04.

There's a step that runs `quilt push` and `quilt refresh` to address the _fuzzy_ patches that Debian packaging would otherwise outright refuse. It seems that this _fuzzy_ hunk just gets removed from the file which is why the error occurs after it's compiled. This might be causing the other patches (CSD) to also break.

https://github.com/lah7/gtk3-classic/blob/07572a74f02558b716890dd30398fff6e2e6aaef/popovers__file-chooser-list.patch#L82-L94

**Comment by lah7** [comment]
Fixed the problematic patch b62bcaa706ef449f6242c70096cb6205a6714d56 by re-creating it. [I also documented how to use quilt](https://github.com/lah7/gtk3-classic/wiki/Creating-and-Managing-Patches) for future reference. New packages are now being built on Launchpad.
